package.json

```diff
@@ -31,6 +30,6 @@
   },
   "dependencies": {
     "@types/react": "^15.0.21",
```

Comment:
um not quite it.
So if you look at debounce,
we have
```
export default function debounce(wait = 500, options?: DebounceSettings) {
 return function(target: any, key: string, descriptor: PropertyDescriptor) {
```

If any of the imported types appear as the parameters of the exported function or any of the parameters of the functions being returned, we need to make it available as a dependency (and not a dev dependency)